<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TotalCommanderClass</class>
 <widget class="QMainWindow" name="TotalCommanderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TotalCommander</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TotalCommander.qrc">
    <normaloff>:/Resources/resources/commander_one_macos_bigsur_icon_190276.ico</normaloff>:/Resources/resources/commander_one_macos_bigsur_icon_190276.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbOK">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbMisha">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Misha</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbKolia">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kilia</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbSvieta">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sveta</string>
    </property>
   </widget>
   <widget class="QLabel" name="lQuestion">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>What's your name?</string>
    </property>
   </widget>
   <zorder>lQuestion</zorder>
   <zorder>pbOK</zorder>
   <zorder>rbMisha</zorder>
   <zorder>rbKolia</zorder>
   <zorder>rbSvieta</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>373</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TotalCommander.qrc"/>
 </resources>
 <connections/>
</ui>
